Packs/CrowdStrikeFalconStreamingV2/Integrations/CrowdStrikeFalconStreamingV2/CrowdStrikeFalconStreamingV2.py

```diff
@@ -109,11 +108,9 @@ async def _http_request(self, method, url_suffix, full_url=None, headers=None, a
                     )
                     time.sleep(time_to_wait)
```

Comment:
Shouldn't we be using the asycio sleep version as we are in a async function?

Reply:
yea, changed